interview-questions.md

```diff
@@ -9,41 +9,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: In my experience 'this' in JavaScript is used to call on a key value in a class.
 
-  Researched answer:
+  Researched answer: 'this' is a JavaScript keyword that refers to the object it belongs to.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a library in JavaScript and is used in front end development. From my understanding we use react to make applications more dynamic.
 
-  Researched answer:
+  Researched answer: React is JavaScript library for creating user interfaces. We use it to build small parts and then join them together to form entire applications. 
```

Comment:
Totally!  React is so versatile with being able to create custom components and reuse them over and over again.  Not to mention the virtual DOM tracking changes and refreshing only the needed changes and not the entire page.